### ISSUE TITLE
SSH-ing into Jenkins container fails

### ISSUE THREAD

**Issue by bleggett** [tmobile/jazz-installer#142]
When I pick scenario 3 (Dockerized jenkins and Gitlab), install fails repeatedly at this point:

```
Granting permissions to other users to pip install....Completed
['18.222.73.235:8081', 'admin', '8dabee45bb234a96bcd2a7c594acf15f', '18.222.73.235', 'root', '2200', 'sg-852143ee', 'subnet-aa5d53d2']
Unable to SSH into the Jenkins instance! Is the jenkinskey key or the username valid? Jenkins IP: 18.222.73.235 Username: root Keypath: /home/centos/jazz-installer/installscripts/dockerfiles/jenkins//jenkinskey.pem Port: 2200
```

Originally this just logged the failure and did not stop script execution, I added the `sys.exit` since there's no point in continuing if this fails. I also added some logging to see why the SSH call (copied below)using paramiko fails

```
def check_jenkins_sshuser_valid(parameter_list, port_number, keypath):
    """
        Check if the ssh login name is a user
    """
    jenkins_server_public_ip = parameter_list[3]
    jenkins_server_ssh_login = parameter_list[4]
    keyfile = keypath + "/jenkinskey.pem"
    try:
        ssh = paramiko.SSHClient()
        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
        ssh.connect(jenkins_server_public_ip, username=jenkins_server_ssh_login, key_filename=keyfile, port=port_number)
    except:
        sys.exit("Unable to SSH into the Jenkins instance! Is the jenkinskey key or the username valid? Jenkins IP: %s Username: %s Keypath: %s Port: %s" %(jenkins_server_public_ip, jenkins_server_ssh_login, keyfile, port_number,))
```

The ssh key is in the correct place, but I can verify that the paramikio ssh failed. Looking in to this. 

When dealing with containers we shouldn't even need to SSH in in most cases, it's a bit roundabout, we can just `docker exec` and skip the part where we have to copy ssh keys around.

**Comment by bleggett** [comment]
Looks like the script (which is running on the docker host machine) is actually getting the public IP of the docker host machine, and ssh-ing from the machine its running on back into the machine its running on. 

But this will fail unless the docker host machine's AWS security group is configured to allow SSH connections on port 2200, which it isn't by default.

The Terraform scripts will add this rule it looks like, but they run _after_ we do a sanity check to see if the user can SSH into the instance, so the sanity check will always fail, which doesn't help us.

**Comment by bleggett** [comment]
Fixed in #149
